website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +76,147 @@ const columns = [
+        description: 'F&B Minimum Per Contract',
+        sortable: true,
+        width: 200,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['F&B Minimum Per Contract']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'area',
+        headerName: 'SQFT per Event',
+        description: 'SQFT per Event',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['SQFT per Event']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type',
+        headerName: 'Type',
+        description: 'Event Type',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['event_type']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'type-account',
+        headerName: 'Type-account',
+        description: 'Event Type account',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['type_account']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'totalDays',
+        headerName: 'Total Days (In-Out)',
+        description: 'Total Days (In-Out)',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['total_days']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isTelecom',
+        headerName: 'isTelecom',
+        description: 'isTelecom',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_telecom']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'isAudio',
+        headerName: 'isAudio',
+        description: 'isAudio',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['is_audio']}`
+            } catch (error) {
+                return '';
+            }
+        }
+    },
+    {
+        field: 'eventDaynumber',
+        headerName: 'EventDayNumber',
+        description: 'EventDayNumber',
+        sortable: true,
+        width: 175,
+        valueGetter: (params) => {
+            try {
+                return `${params.getValue('prediction_parameters')['event_daynumber']}`
```

Comment:
This piece of data does not exist in the `prediction_parameters` object. If you are attempting to do what I outlined in the previous comments, you will need to calculate this manually.